data/src/main/kotlin/com/globant/data/repositories/MarvelCharacterRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.globant.data.repositories
+
+import com.globant.domain.entities.MarvelCharacter
+import com.globant.domain.repositories.MarvelCharacterRepositoryContract
+
+class MarvelCharacterRepository : MarvelCharacterRepositoryContract {
```

Comment:
I would change MarvelCharacterRepositoryContract to MarvelCharacterRepository only